modules/page-memory/src/test/java/org/apache/ignite/internal/pagememory/persistence/store/FilePageStoreManagerTest.java

```diff
@@ -440,13 +449,68 @@ void testFullyRemovePartitionOnStart() throws Exception {
         );
     }
 
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 4, 10, 25, 50})
```

Comment:
Let's add a sane time out (like `@Timeout(value = 1, unit = MINUTE)`) just in case as racers use potentially infinite loops (if something gets broken in the future, they could hang forever)

Reply:
Why not